monitor/README.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: pattern
+title: Monitor
+folder: monitor
+permalink: /patterns/monitor/
+categories: Concurrency
+tags:
+ - Thread-safe
+ - Mutual exclusion
+---
+
+## Intent
+Monitor pattern is used to create thread-safe object . In the concurrent application to prevent conflict between threads .
+## Explanation
+
+In plain words
+
+> Monitor pattern is used when we have critical section or race condition on shared resource in your application so we have to use mutual exclusion to solve this problem ! one of the ways to achieve mutual exclusion is Monitor pattern that invented by Tony Hoare.
+
+Wikipedia says
+
+> In concurrent programming (also known as parallel programming), a monitor is a synchronization construct that allows threads to have both mutual exclusion and the ability to wait (block) for a certain condition to become false. Monitors also have a mechanism for signaling other threads that their condition has been met. 
+
+## Class diagram
+![alt text](./etc/monitor.urm.png "Monitor class diagram")
```

Comment:
The file is missing

Reply:
i know !
but i thought you create UML image !  because i dont know with which tool must make UML like your form .

Reply:
I would request you to go through the **[Wiki](https://github.com/iluwatar/java-design-patterns/wiki)** section to understand everything as every contributing guideline is very deeply elaborated over there.

If you still find some issues or need help in some matters, feel free to mention the fact.

Thanks.

Reply:
i read Wiki . but idont know how to create class diagram exactly ! 
is UML class diagram necessary ? because the pattern does not have any important relations !

Reply:
As mentioned in the docs it will be in the target folder

Reply:
Ok !
Can you help me to create UML class diagram?